tests/post_condition/dav1d.c

```diff
@@ -2,18 +2,27 @@
 RUN: cat dav1d_call_gates_1.ld | FileCheck --check-prefix=LINKARGS %s
 */
 
-#include <ia2_test_runner.h>
 #include "dav1d.h"
+#include <ia2_test_runner.h>
+#include <signal.h>
+
+bool corrupt_stride = false;
```

Comment:
Doesn't this need to be marked as shared? It's accessed from another compartment.

Reply:
Yup, good catch.

Reply:
The lib compartment is using pkey 0 so it's fine.

Reply:
How do I get the lib compartment to not use pkey 0?  `#define IA2_COMPARTMENT 2` in `dav1d.c` doesn't seem to fix it either.

Reply:
Okay I think I figured it out.  I didn't realize how to define the test in cmake properly before.